### PR TITLE
[Distributed] add cluster setup example doc

### PR DESCRIPTION
**Pull request by LebronAl**

### CODE REVIEW

File: docs/UserGuide/Cluster/Cluster-Setup-Example.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+## Example of distributed scaffolding for 3 nodes and 3 replicas
+
+Suppose we need to deploy the distributed IoTDB on three physical nodes, A, B, and C, whose public network IP is a_public_IP, b_public_IP, and c_public_IP, and private network IP is a_private_IP, b_private_IP, and c_private_IP.
+Note: If there is no public network IP or private network IP, both can be set to the same, just need to ensure that the client can access the server.
+
+The operation steps are as follows:
+
+1. Make sure ports 6667, 9003, 9004, 40010, 40011 and 31999 are open on all three nodes.
+2. Use 'mvn clean package -pl cluster -am -DskipTests' to compile the distributed module.
+3. Send the generated package (IOTDB-cluster-0.12.0-Snapshot) to all servers.
```
Comment (mychaow) [inline docs/UserGuide/Cluster/Cluster-Setup-Example.md:43]: IOTDB -> iotdb?

**Review by neuyilan — APPROVED** [review]
LGTM

**Review by wangchao316 — APPROVED** [review]
LGTM

File: docs/UserGuide/Cluster/Cluster-Setup-Example.md (hunk 1 above)
Comment (LebronAl) [reply docs/UserGuide/Cluster/Cluster-Setup-Example.md]: got it

**Review by mychaow — APPROVED** [review]
LGTM

**Pull request merged**